docs/quickstart/readme.md

```diff
@@ -0,0 +1,310 @@
+
+<img src="img/create_scene.png" width = 50%/>
+</details>
+
+If everything went alright, entering the scene selector by pressing the icon next to the scene name should look like this:
+
+<details>
+<summary>Scene selector</summary>
+
+<img src="img/scenes.png" width = 50%/>
+</details>
+
+If you switched around scenes to see if everything worked, make sure to switch back to the `Library` scene now.
+
+Press `N`. This will open a side window. In it, click on the tab `Blenvy`. This is the main window through which you will interact with Blenvy.
+
+<details>
+<summary>Blenvy menu</summary>
+<img src="img/blenvy_menu.png" width = 50%/>
+</details>
+
+You will see that two options are highlighted red because they are not yet configured.
+
+For `level scenes`, click on the selector, select `World` and hit `+`. For `library scenes`, select `Library` and again hit `+`.
+
+Your menu should now look like this:
+
+<details>
+<summary>Correctly setup Blenvy</summary>
+<img src="img/blenvy_after_setup.png" width = 50%/>
+</details>
+
+Save your Blender file by pressing `Ctrl + S` on Windows or Linux and `Cmd + S` on macOS. You should now have a new asset under `my_game/assets/levels/World.glb` and a companion file containing metadata.
+
+<details>
+<summary>The newly created level assets</summary>
+<img src="img/level_asset.png" width = 50%/>
+</details>
+Run, run your game with `cargo run`. It may crash because the scene is empty, but don't worry. The point is that now will have generated another new file, namely `assets/registry.json`. This file contains the information about all components exported by your game so that Blenvy can pso them in Blender.
+
+<details>
+<summary>The final file structure for an empty world</summary>
+<img src="img/registry.png" width = 50%/>
+</details>
+
+This concludes the setup portion of the guide.
+
+## Create a blueprint
+
+Alright, let's jump into the actual workflow you'll be using to create your game.
+We will first create an object for the `Player` component we defined earlier.
+While still in the `Library` scene, right-click on the `Scene Collection` in the outliner to the right. Select `New Collection`. Double-click on the new collection that appeared and rename it to `Player`. Click on it to have it selected.
+
+<details>
+<summary>The player collection</summary>
+<img src="img/player_collection.png" width = 50%/>
+</details>
+
+Now, on the upper left of Blender, click on `Add` -> `Mesh` -> `Cube`. This will be our player. We will now add the `Player` component to it. Go to the Blenvy menu we opened earlier. Select the icon in its upper left corner. This is where components are managed.
+
+<details>
+<summary>The component manager</summary>
+<img src="img/component_manager.png" width = 50%/>
+</details>
+
+If your component manager says "Select an object to edit its components", make sure you have the *Cube* we just created selected by clicking on it in the 3D viewport.
```

Comment:
is there a specific reason for adding the component to the cube and not the collection/blueprint itself ?

Reply:
Not really, it's just muscle memory from the pre-Blenvy days. Should I add it to the collection instead?

Reply:
Yes, I would change it : adding components to the blueprint has a different meaning/ use case then adding it to a mesh inside the blueprint for example (your Avian tutorial makes it even clearer, adding a component to the mesh means you want to add some feature specific to the mesh).
I don't know if I am making much sense 😄
Don't bother changing any relevant screenshots though, I'll take care of that :)

Reply:
I think I replaced all of the mentions in the text: [`4c5cebd` (#213)](https://github.com/kaosat-dev/Blenvy/pull/213/commits/4c5cebd00a167fd9105a88af0605a67f1743c877)